doc/src/guides/solving/solving-guidance.md

```diff
@@ -0,0 +1,245 @@
+# Solving Guidance
+
+These guidelines apply to many types of solving.
+
+## Numeric Solutions
+
+### Equations With no Analytical Solution
+
+The vast majority of arbitrary nonlinear equations are not analytically
+solvable. The classes of equations that are solvable are basically:
+1. Linear equations
+2. Polynomials, except where limited by the [Abel-Ruffini
```

Comment:
Perhaps a mention to Groebner basis reduction for systems of polynomial equations? @oscarbenjamin is always dazzling me with demonstrations of this on SO. I wish I understood more but a link to that function might help people be aware of this technique.